uint16 Distance
uint8 Intensity
